ci/prod/deploy-pipeline.yaml

```diff
@@ -38,17 +38,17 @@ spec:
 
     - name: release
       type: github-release
-      icon: file-document
+      icon: tag
       source:
         <<: *github_source
 
-    - name: daily
+    - name: nightly
       type: time
       icon: update
       source:
-        interval: 12h
-        start: 8:00 AM
-        stop: 8:00 PM
+        interval: 1h
```

Comment:
it's a super minor thing, but I don't think this `interval:` line is required: https://github.com/concourse/time-resource#trigger-once-within-time-range

That said, happy to leave it be

Reply:
I'm happy to leave it. It's the same mechanism we're using for the daily build and it works fine so no reason to change it